lib/models.py

```diff
@@ -355,7 +355,7 @@ def approved_and_ranked(self, proposal_quorum, next_superblock_max_budget):
         return ranked
 
     @classmethod
-    def expired(self, dashd):
+    def expired(self):
```

Comment:
It doesn't use the `dashd` argument and wasn't called with it initially, the method signature was the only thing wrong with it, so cleaning this up fixed the issue. It was erroring with some message about a missing positional argument before this change.